compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -3191,151 +3191,99 @@ OMR::Power::CodeGenerator::addMetaDataForLoadAddressConstantFixed(
       int16_t typeAddress,
       intptrj_t value)
    {
+   if (value == 0x0)
+      return;
+
+   if (typeAddress == -1)
+      typeAddress = TR_FixedSequenceAddress2;
 
-   bool doAOTRelocation = true;
+   TR_FixedSequenceKind seqKind;
    TR::Compilation *comp = self()->comp();
 
    if (tempReg == NULL)
+      seqKind = fixedSequence1;
+   else
+      seqKind = fixedSequence5;
```

Comment:
Consider the more succinct:

`TR_FixedSequenceKind seqKind = tempReg ? fixedSequence5 : fixedSequence1;`